Documentation/configuration/index.rst

```diff
@@ -19,3 +19,35 @@ Core Agent
    sctp
    vlan-802.1q
    argocd-issues
+
+Verify Signed Container Images
+------------------------------
+
+Before you begin
+~~~~~~~~~~~~~~~~
+
+You will need to have the following tools installed:
+
+    - cosign (`install guide`_)
+    - curl (often provided by your operating system)
+
+.. _`install guide`: https://docs.sigstore.dev/cosign/installation/
+
+Verifying image signatures
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+For a complete list of images that are signed please refer to `Releases`_.
+
+Let's pick one image from this list and verify its signature using the cosign verify command:
+
+.. code-block:: shell-session
+
+    $ COSIGN_EXPERIMENTAL=1 cosign verify quay.io/cilium/cilium:v1.x.x
```

Comment:
Thanks for mentioning what `COSIGN_EXPERIMENTAL` is for. Further question, do you know the `_EXPERIMENTAL` suffix implies in terms of lifetime for the feature? Is this something that is not completely stable yet, or is it an experiment that could be removed from `cosign` in a future version by its maintainers if not deemed interesting?